### PR TITLE
Utils update

### PR DESCRIPTION
**Pull request by npmitche**
Update Utils to better align and be compatible with the updates done in PR #1832 

Split PR into two commits: First modifies the base files. Second modifies the util commands to align.

### CODE REVIEW

File: chipsec/utilcmd/lock_check_cmd.py [hunk 1]
```diff
@@ -75,9 +77,15 @@
         parser_check = subparsers.add_parser('lock', parents=[parser_lockname])
         parser_check.set_defaults(func=self.check_lock)
 
-        parser.parse_args(self.argv[2:], namespace=self)
+        parser.parse_args(self.argv, namespace=self)
 
-        return True
+    def set_up(self) -> None:
+        CONSISTENCY_CHECKING = True # TODO: Not doing anything.
```
[the remark this answers is not in the document]
Comment (npmitche) [reply chipsec/utilcmd/lock_check_cmd.py:83]: Will fix in a future update. Simply copied line from old `def run()` function.

File: chipsec_util.py [hunk 2]
```diff
@@ -162,40 +162,49 @@ def parse_switches(self) -> None:
...
-        req_driver = comm.requires_driver()
-        if req_driver and self._no_driver:
-            self.logger.log("Cannot run without driver loaded", level.ERROR)
-            sys.exit(ExitCode.OK)
+        comm = self.commands[self._cmd](self._cmd_args, cs=self._cs)
+        comm.parse_arguments()
+        reqs = comm.requirements()
+        if reqs.load_driver() and self._no_driver:
+            self.logger.log("Cannot run command without a driver loaded.", level.ERROR)
+            return ExitCode.OK
```
Comment (npmitche) [inline chipsec_util.py:173]: switch to ExitCode.ERROR

File: chipsec_util.py [hunk 3]
```diff
@@ -162,40 +162,49 @@ def parse_switches(self) -> None:
...
-    
-            
 
-        self.logger.log("[CHIPSEC] Executing command '{}' with args {}\n".format(self._cmd, self.argv[2:]))
+        self.logger.log(f"[CHIPSEC] Executing command '{self._cmd}' with args {self._cmd_args}\n")
+        
+        try:
+            comm.set_up()
+        except Exception as msg:
+            self.logger.log(msg)
```
Comment (iherna2) [inline chipsec_util.py:199]: Would log_error() be more appropriate?

File: chipsec_util.py [hunk 4]
```diff
@@ -162,40 +162,49 @@ def parse_switches(self) -> None:
...
+            self.logger.log("Cannot run command without a driver loaded.", level.ERROR)
+            return ExitCode.ERROR
+            
+        if reqs.load_config() and not self._load_config:
+            self.logger.log("Cannot run command without a config loaded. Please run with -p and/or --pch if needed.", level.ERROR)
+            return ExitCode.ERROR
 
         try:
-            self._cs.init(self._platform, self._pch, self._helper, ((not self._no_driver) and req_driver), self._load_config, self._ignore_platform)
+            self._cs.init(self._platform, self._pch, self._helper, ((not self._no_driver) and reqs.load_driver()), (self._load_config and reqs.load_config()), self._ignore_platform)
```
Comment (iherna2) [inline chipsec_util.py:180]: Do we have precedence between the module declared requirements and the command line arguments? We already did a sanity check in 172 and 175, so here canwe  just use reqs.load_driver() ?

**Review by iherna2 — COMMENTED** [review]
There are a several touched files without copyright year update

**Comment by npmitche** [comment]
> There are a several touched files without copyright year update

As long as it has a copyright year, we don't have to update it anymore.

**Review by BrentHoltsclaw — APPROVED** [review]

**Review by iherna2 — APPROVED** [review]

**Review by dscott90 — APPROVED** [review]

**Pull request merged**